tests/js/environment.js

```diff
@@ -1,23 +1,28 @@
 var fluid = require("infusion");
 
-fluid.defaults("gpii.pouch.tests.environment", {
+fluid.defaults("gpii.test.pouch.environment", {
     gradeNames: ["fluid.test.testEnvironment"],
     port:       6792,
     baseUrl:    "http://localhost:6792/",
     testUrl:    "/sample/",
     events: {
-        constructServer: null,
-        onStarted: null
+        constructFixtures: null,
+        onHarnessReady: null,
+        onFixturesConstructed: {
+            events: {
+                onHarnessReady: "onHarnessReady"
+            }
+        }
```

Comment:
I refactored to avoid the double-relaying, and to isolate the default settings from the environment and harness.

I did not use the `pouchConfig` name, which is already used within the pouch grade itself, and for a different purpose.  I will add the docs shortly to clarify the total range of options and we can discuss whether additional work is needed.